app/Nova/Resources/Auth0Client.php

```diff
@@ -108,27 +115,51 @@ public function actions(NovaRequest $request): array
     {
         return [
             App::make(ActivateAuth0Client::class)
-                ->showOnDetail()
-                ->showInline()
+                ->exceptOnIndex()
                 ->confirmText('Are you sure you want to activate this client?')
                 ->confirmButtonText('Activate')
                 ->cancelButtonText("Don't activate")
-                ->canRun(function ($request, $model) {
-                    /** @var ActivateAuth0ClientGuard $guard */
-                    $guard = App::make(ActivateAuth0ClientGuard::class);
-                    return $guard->canDo($model->toDomain());
-                }),
+                ->canSee(fn (Request $request) => $this->canActivate($request, $this->resource))
+                ->canRun(fn ($request, $model) => $this->canActivate($request, $model)),
```

Comment:
You could also type the function arguments of the canRun?
There are multiple instances, where they're not typed